### ISSUE TITLE
Cannot use flags --name

### ISSUE THREAD

**Issue by FahriDevZ** [jmcdo29/nest-commander#24]
This is first time i use package commander, and i don't understand where this problem coming.
And how to i use flags --name?

error log:
```
~/TestNestCommander$ node ./dist/main.js
(node:30) UnhandledPromiseRejectionWarning: Error: option 'name' clashes with existing property 'name' on Command
- call storeOptionsAsProperties(false) to store option values safely,
- or call storeOptionsAsProperties(true) to suppress this check,
- or change option name

Read more on https://git.io/JJc0W
    at Command._checkForOptionNameClash (/home/runner/TestNestCommander/node_modules/commander/index.js:472:13)
    at Command._optionEx (/home/runner/TestNestCommander/node_modules/commander/index.js:499:10)
    at Command.option (/home/runner/TestNestCommander/node_modules/commander/index.js:622:17)
    at CommandRunnerService.setUpCommander (/home/runner/TestNestCommander/node_modules/nest-commander/dist/command-runner.service.js:51:28)
    at CommandRunnerService.onModuleInit (/home/runner/TestNestCommander/node_modules/nest-commander/dist/command-runner.service.js:32:14)
    at process._tickCallback (internal/process/next_tick.js:68:7)
    at Function.Module.runMain (internal/modules/cjs/loader.js:834:11)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
(node:30) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:30) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

Example: https://replit.com/@fahri5567/TestNestCommander

**Comment by jmcdo29** [comment]
Lots of really good discussion on this [over here](https://github.com/tj/commander.js/issues/183) and in related issues. Looks like there's a better API than what I'm already using, so I'll get the code updated here soon and add in some tests to ensure it keeps working. Thanks for bringing this up

**Comment by jmcdo29** [comment]
@FahriDevZ did you mean to close this back out?

**Comment by FahriDevZ** [comment]
Wow, thanks for your quick reply. Due this issue I temporarily only use single dash :(

**Comment by FahriDevZ** [comment]
Maybe to solve this problem should upgrade the commander.
I've done test `test:int` with args --name and it worked, however there seems to be a problem with the `Hello Command` test 🤔

**Comment by jmcdo29** [comment]
Looks like this is actually already fixed in commander v7, which I'll have the next version of nest-commander use under the hood.

**Comment by jmcdo29** [comment]
Okay, this should be fixed in version 1.3.0
